factfinder/data/acs/2010/median.json

```diff
@@ -601,6 +617,8 @@
     },
     "mdvl": {
         "design_factor": 1.4,
+        "top_coding": false,
```

Comment:
I believe this should be true, but it looks like the top coding threshold is not the lower boundary of the top bin. I'm not sure how to implement this one.